src/components/CertificateDropZone/Views/DefaultView.js

```diff
@@ -11,7 +11,10 @@ const View = ({ hover, accept }) => (
   >
     <div className={css["image-container"]}>
       <i>
-        <img src="/static/images/dropzone/dropzone_illustration.svg" />
+        <img
+          alt="Dropzone"
```

Comment:
```suggestion
          alt=".opencert Dropzone"
```

Might want to consider above, or a variation. Reason is to help people figure out "how to open .opencert".